### PR TITLE
Fix media selection for none images

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | part of #2654 (1,2)
| License | MIT

#### What's in this PR?

Media selection dont work for none images. All medias are shown as selected medias.

#### Why?

Bugs ...

### CODE REVIEW

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/overlay/main.js [hunk 1]
```diff
@@ -236,7 +236,7 @@ define([
 
             this.sandbox.emit('husky.datagrid.' + this.options.instanceName + '.selected.update', ids);
             this.sandbox.emit('husky.datagrid.' + this.options.instanceName + '-selected.url.update', {
-                ids: ids.join(',') || ' '
+                ids: ids.join(',') || ' ' // else all are selected
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/overlay/main.js]: What do you mean by this comment?

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/overlay/main.js [hunk 2]
```diff
@@ -436,9 +441,7 @@ define([
                             url: [
                                 '/admin/api/media?locale=', this.options.locale,
                                 '&fields=id,thumbnails,title,size',
-                                '&orderBy=media.created&orderSort=DESC&ids=', _.map(this.items, function(item) {
-                                    return item.id;
-                                }).join(',')
+                                '&orderBy=media.created&orderSort=DESC&ids=', ids ? ids : ' ' // else all are loaded
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/overlay/main.js]: Same as above, and write `desc` in lowercase

File: src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/overlay/main.js (hunk 1 above)
Comment (alexander-schranz) [inline src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/overlay/main.js]: as shown in the issue: https://github.com/sulu/sulu/issues/2654 when you open the media selection overlay and no media is selected all are shown as selected.

**Comment by alexander-schranz** [comment]
@danrot did remove the old hack with `ids=whitespace` by add a limit for this I needed to update the ListPresentation because of a Divided by Zero error.

**Pull request merged**